src/Calculator/GraphingSettings.xaml

```diff
@@ -0,0 +1,136 @@
+<Page
+    x:Class="CalculatorApp.Controls.GraphingSettings"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:CalculatorApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    DataContextChanged="GraphingCalculator_DataContextChanged"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Loaded="OnLoaded"
+    >
+    <Page.Resources>
```

Comment:
I don't think many of these colors will work with high contrast. We should have a ResourceDictionary for each theme, check out App.xaml for an example on handling high contrast brushes.